shared/js/initApp.js

```diff
@@ -1,9 +1,21 @@
+// TODO (brent) - way too many globals
+// TODO (brent) - I wonder if we should sub-namespace dashboard
+/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Blockly, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga*/
```

Comment:
Cool. Obviously not something we need to do right away, it seems like there's probably opportunity to make things a little cleaner without needing to investing a lot of work :)